src/backend/app/auth/temp_auth.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2022, 2023 Humanitarian OpenStreetMap Team
+#
+# This file is part of FMTM.
+#
+#     FMTM is free software: you can redistribute it and/or modify
+#     it under the terms of the GNU General Public License as published by
+#     the Free Software Foundation, either version 3 of the License, or
+#     (at your option) any later version.
+#
+#     FMTM is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU General Public License for more details.
+#
+#     You should have received a copy of the GNU General Public License
+#     along with FMTM.  If not, see <https:#www.gnu.org/licenses/>.
+#
+
+"""Temporary Auth methods related to field users."""
+from fastapi import APIRouter, Request
+from fastapi.responses import JSONResponse
+
+from app.config import settings
+
+router = APIRouter(
+    prefix="/temp",
+    tags=["temp-auth"],
+    responses={404: {"description": "Not found"}},
+)
+
+
+@router.get("/login/")
+async def temp_login(
+    request: Request,
```

Comment:
One last thing: I think we should take an extra param `email` here. 

Thus implementation will work for now. 

When we have an email service included, maybe we ads the extra step to require verification by clicking a button in the users email inbox.

Reply:
just an extra param for now?